src/main/java/org/kohsuke/github/Preview.java

```diff
@@ -9,9 +9,10 @@
 /**
  * Indicates that the method/class/etc marked maps to GitHub API in the preview period.
  * <p>
- * These APIs are subject to change and not a part of the backward compatibility commitment. Always used in conjunction
- * with 'deprecated' to raise awareness to clients. In addition, it's advised to update the targets documentation to
- * signify that the deprecation is required until preview feature being used is promoted to stable.
+ * These APIs are subject to change and not a part of the backward compatibility commitment.
+ *
+ * It's advised to update the targets documentation with text signify that the preview feature being used is promoted to
+ * stable.
```

Comment:
this doesn't make sense to me, I assume you mean that there should be a warning added to the javadoc?

Reply:
```suggestion
 * It is advised to update the target's documentation with text indicating that a preview feature being used.
```